samples/msal-node-samples/auth-code-key-vault/index.js

```diff
@@ -69,48 +68,117 @@ function msalApp(thumbprint, privateKey) {
         });
     });
 
-    app.post('/redirect', (req, res) => {
+    app.post("/redirect", (req, res) => {
         const tokenRequest = {
             code: req.body.code,
             scopes: ["user.read"],
             redirectUri: REDIRECT_URI,
         };
 
         cca.acquireTokenByCode(tokenRequest).then((response) => {
-            res.status(200).send('Congratulations! You have signed in successfully');
+            res.status(200).send("Congratulations! You have signed in successfully");
         }).catch((error) => {
             res.status(500).send(error);
         });
     });
 
     app.listen(SERVER_PORT, () => {
-        console.log(`Msal Node Auth Code Sample app listening on port ${SERVER_PORT}!`)
+        console.log(`Msal Node Auth Code Sample app listening on port ${SERVER_PORT}!`);
     });
 }
 
 async function main() {
     let certResponse;
     try {
-        // Grab the certificate thumbprint
+        // Grab the certificate thumbprint from the Key Vault
         certResponse = await certClient.getCertificate(CERTIFICATE_NAME);
     } catch (error) {
-        console.log(error);
+        console.log("Error retrieving certificate from Key Vault:", error);
```

Comment:
We discussed about exception handling before. We should not catch and log exceptions. You are breaking an important contract between JS and Express. 

The correct pattern is to let exceptions bubble up. Execution cannot continue if you don't have a cert.

If you want to improve the exception message, throw a new exception which encapsualtes the existing one. 

```js
catch (error) {
console.error("Error retrieveing cert from KeyVault: ", error.Message);
throw error;
}
```